getusermedia.html

```diff
@@ -2864,17 +2864,23 @@ <h2>Methods</h2>
                 <li>
                   <p>Let <var>p</var> be a new promise.</p>
                 </li>
+                <li>
+                  <p>Let <var>proceed</var> be the result of
+                  [=device enumeration can proceed=].</p>
+                </li>
+                <li>
+                  <p>Let <var>document</var> be the [=relevant global object=]'s
+                  [=associated `Document`=].</p>
+                </li>
                 <li>
                   <p>Run the following steps in parallel:</p>
                   <ol>
                     <li>
-
-                      <p>Let <var>document</var> be the [=relevant global object=]'s
-                      [=associated `Document`=].</p>
-                    </li>
-                    <li>
-                      <p>The [=User Agent=] MUST wait to proceed to the next step until
-                      [=device enumeration can proceed=] is <code>true</code>.</p>
+                      <p>If <var>proceed</var> is `false`, the [=User Agent=]
```

Comment:
Maybe this better describes the situation 
```suggestion
                      <p>While <var>proceed</var> is `false`, the [=User Agent=]
```